### DISCUSSION TITLE
"PI Pico W" bluetooth support

### DISCUSSION THREAD

**Discussion by oke24** [grblHAL/RP2040#61] [Ideas]
In Pico SDK 1.5 there is now bluetooth support implemented as new feature.
( https://github.com/raspberrypi/pico-sdk/releases/tag/1.5.0 )
It would be great if the RP2040 driver supports also the Pico W onboard BT capabilities. 
e.g. use mobile phones bluetooth with grbl controller app.

**Comment by terjeio** [comment]
It is already on my todo list.

**Reply by terjeio** [reply]
Pico W Bluetooth support has now been added.

**Reply by oke24** [reply]
Wonderful ! Works "out of the box" with grblController App on Android - thank you very much !
